package.json

```diff
@@ -21,7 +21,9 @@
   "devDependencies": {
     "depcheck": "^0.6.4",
     "eslint": "^3.5.0",
-    "fixpack": "^2.3.1"
+    "fixpack": "^2.3.1",
+    "npm-run-all": "^3.1.0",
```

Comment:
I'm starting to like this **npm-run-all** since it avoids silly long chained `&&` scripts.
Plus, by breaking up npm scripts, I can do `$ npm run lint:eslint -- --fix` to just run the one script and autofix errors.